src/OSPSuite.Core/Domain/Services/SimModelManagerBase.cs

```diff
@@ -44,9 +44,10 @@ protected void FinalizeSimulation(Simulation simModelSimulation)
          simModelSimulation.FinalizeSimulation();
       }
 
-      protected Simulation CreateSimulation(string simulationExport)
+      protected Simulation CreateSimulation(string simulationExport, Action<SimulationOptions> configureSimulationAction = null)
```

Comment:
@Yuri05 I have added a configuration method that can be called. That way, we do not have to add one flag for each possible option. Instead caller can decide what to do

Reply:
Nice!